qa/workunits/rbd/rbd_groups.sh

```diff
@@ -146,6 +166,15 @@ check_snapshot_in_group()
     list_snapshots $group_name | grep $snap_name
 }
 
+check_snapshots_count_in_group()
+{
+    local group_name=$1
+    local snap_name=$2
+    local expected_count=$3
+    local actual_count=$(list_snapshots $group_name | grep -c $snap_name)
```

Comment:
The exit code from `$(...)` is not propagated by `local`, the fix is to split the declaration and the assignment:
```suggestion
    local actual_count
    local_count=$(list_snapshots $group_name | grep -c $snap_name)
```